master/getting-started/kubernetes/index.md

```diff
@@ -51,40 +51,70 @@ the host. Instead, continue directly to the
    sudo cp -i /etc/kubernetes/admin.conf $HOME/.kube/config
    sudo chown $(id -u):$(id -g) $HOME/.kube/config
    ```
-   
-1. Install {{site.prodname}} and a single node etcd with the following command.
+
+1. Install an etcd instance on the master with the following command.
 
    ```
    kubectl apply -f \
-   {{site.url}}/{{page.version}}/getting-started/kubernetes/installation/hosted/kubeadm/1.7/calico.yaml
+   {{site.url}}/{{page.version}}/getting-started/kubernetes/installation/hosted/etcd.yaml
    ```
-   
-   > **Note**: You can also 
-   > [view the YAML in a new tab]({{site.url}}/{{page.version}}/getting-started/kubernetes/installation/hosted/kubeadm/1.7/calico.yaml){:target="_blank"}.
-   {: .alert .alert-info}
 
    You should see the following output.
 
    ```
-   configmap "calico-config" created
    daemonset "calico-etcd" created
    service "calico-etcd" created
-   daemonset "{{site.noderunning}}" created
-   deployment "calico-kube-controllers" created
-   clusterrolebinding "calico-cni-plugin" created
-   clusterrole "calico-cni-plugin" created
-   serviceaccount "calico-cni-plugin" created
-   clusterrolebinding "calico-kube-controllers" created
-   clusterrole "calico-kube-controllers" created
+   ```
+
+
+   > **Note**: This command installs a single-node etcd on the master for use by Calico. In a production environment,
```

Comment:
We have been trying to use Notes just below the manifests which allow people to view the manifest in a new tab. We make it pretty clear in the introduction that the procedure is not suitable for a production install. For these reasons, I would suggest replacing this note with the note that lets people view the manifest, and including notes of this kind below each manifest.